src/DemoSite/Editor.js

```diff
@@ -123,12 +123,13 @@ const Editor = ({ onOpenAnnotator, lastOutput }: any) => {
             className="button"
             variant="outlined"
             disabled={Boolean(currentError)}
-            onClick={() =>
-              onOpenAnnotator(
-                selectedExample === "Custom"
-                  ? loadSavedInput()
-                  : examples[selectedExample]
-              )
+            onClick={() =>{
+                onOpenAnnotator(
+                  selectedExample === "Custom"
+                    ? loadSavedInput()
+                    : examples[selectedExample]
+                )
```

Comment:
this should be run through prettier

Reply:
#34 9fd6df7577886b2511c947c04aa0eb11126c56a8